extensions/Worker.Extensions.EventHubs/release_notes.md

```diff
@@ -4,8 +4,7 @@
 - My change description (#PR/#issue)
 -->
 
-### Microsoft.Azure.Functions.Worker.Extensions.EventHubs 6.3.3
+### Microsoft.Azure.Functions.Worker.Extensions.EventHubs 6.3.5
```

Comment:
nit Question: Do we usually jump/skip versions like this? Should we use 6.3.4 ?

Reply:
Just to keep it consistent with Webjobs extension version. Not a strong opinion on this. A similar suggestion form Jacob in previous PR - https://github.com/Azure/azure-functions-dotnet-worker/pull/2626#discussion_r1692052253